tests/components/Dropdown.stories.tsx

```diff
@@ -43,11 +43,15 @@ export const Primary = (args: DropdownProps) => {
   );
 };
 
-export const DropdownExpanded = Primary.bind({});
-DropdownExpanded.play = ({ canvasElement }) => {
+function clickTextbox(canvasElement) {
```

Comment:
nit: can this function be moved to the bottom of the file instead of having it split up the stories?